includes/class-freemius.php

```diff
@@ -10037,7 +10495,12 @@ public static function _uninstall_plugin_hook() {
                     return;
                 }
 
-                $fs->_uninstall_plugin_event();
+                if (
+                    ! $fs->is_clone() &&
+                    ( ! is_object( $fs->_site ) || $fs->is_registered() )

```

Comment:
@fajardoleo it isn't clear why specifically when there's no install context (`! is_object( $fs->_site )`) we should run the uninstall handler.

Reply:
@vovafeldman When there's no install context, it works like before. On the other hand, if there's an install context, it should work like before only if there's a valid user entity.

Reply:
@fajardoleo `it should work like before only if there's a valid user entity` it is not clear to me why. Please add a comment that explains it and why.